doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -1,16 +1,25 @@
 .. _nrf7002dk_nrf5340:
 
-Getting started with nRF7002 DK
-###############################
+Getting started with nRF70 Series
+#################################
 
 .. contents::
    :local:
-   :depth: 2
+   :depth: 4
+
+This section gets you started with your nRF70 Series :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to perform a quick test of your DK.
+
+Supported development boards
+****************************
+
+nRF7002 DK
+==========
 
 The nRF7002 DK (PCA10143) is a single-board development kit for evaluation and development on the nRF7002, a Wi-Fi® companion IC to Nordic Semiconductor's nRF5340 System-on-Chip (SoC) host processor.
 
 Overview
-********
+--------
 
 The nRF7002 is an IEEE 802.11ax (Wi-Fi 6) compliant solution that implements the Wi-Fi physical layer and MAC layer protocols.
```

Comment:
Since we are reviewing this once more. Guess it is nice to have MAC abbreviation also explained (as for physical layer). So medium access control, it should be